wagtail/wagtailadmin/static_src/wagtailadmin/scss/components/_indicator.scss

```diff
@@ -0,0 +1,62 @@
+// =============================================================================
+// Indicator light
+// =============================================================================
+
+
+// =============================================================================
+// Indicator light
+// =============================================================================
+$c-indicator-size: .625em;
+$c-indicator-margin: .25rem;
+
+.c-indicator {
+    display: inline-block;
+    border-radius: 50rem;
+    width: $c-indicator-size;
+    height: $c-indicator-size;
+    margin-top: -.125rem;
+    margin-right: $c-indicator-margin;
+    font-size: 1rem;
+    vertical-align: middle;
+}
+
+// =============================================================================
+// States
+// =============================================================================
+
+$color-state-live: #59b524;
```

Comment:
Seems likely we're going to want to use these colours to replace the main Explorer status tag soon. Could this be moved to the main variables.scss?